main.py

```diff
@@ -10,44 +11,68 @@
 import secrets
 
 
-async def connect():
-    opts = RobotClient.Options(
-        refresh_interval=0,
-        dial_options=DialOptions(credentials=secrets.creds)
-    )
-    return await RobotClient.at_address(secrets.address, opts)
+
+class Robot:
+    UPPER_POSITION = 30
+    LOWER_POSITION = 0
+    WIGGLE_AMOUNT = 5
+
+    async def connect(self):
+        opts = RobotClient.Options(
+            refresh_interval=0,
+            dial_options=DialOptions(credentials=secrets.creds)
+        )
+        self.robot = await RobotClient.at_address(secrets.address, opts)
+        self.servo = Servo.from_robot(self.robot, "servo")
+
+    async def raise_hand(self):
+        await self.servo.move(self.UPPER_POSITION)
+
+    async def lower_hand(self):
+        await self.servo.move(self.LOWER_POSITION)
+
+    async def wiggle_hand(self):
+        for _ in range(3):
+            await self.servo.move(self.UPPER_POSITION + self.WIGGLE_AMOUNT)
+            time.sleep(0.3)
+            await self.servo.move(self.UPPER_POSITION)
+            time.sleep(0.3)
+
+
+@contextlib.asynccontextmanager
+async def makeRobot():
+    robot = Robot()
+    await robot.connect()
```

Comment:
It would be neat to have some kind of `async __init__` function so we could roll these two lines together, but Nicole and I couldn't figure that out. Suggestions welcome, if you've got ideas!

Reply:
Yeah, it turns out that [what I want doesn't work](https://stackoverflow.com/questions/33128325/how-to-set-class-attribute-with-await-in-init), and this is the workaround people usually use.